### PR TITLE
afinfo: add page (en, zh)

### PR DESCRIPTION
**Pull request by garily**
__Audio File Metadata Parser for Darwin__

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/osx/afinfo.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# afinfo
+
+> Audio file metadata parser for OS X.
+> `afinfo --help` or `afinfo -h` for full usage.
```
Comment (einverne) [inline pages/osx/afinfo.md]: `-h` option should not in description. Just move this line as an example.

File: pages/osx/afinfo.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# afinfo
+
+> Audio file metadata parser for OS X.
```
Comment (einverne) [inline pages/osx/afinfo.md]: Is there any document link we can add here?

File: pages/osx/afinfo.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display info of given audio file:
+
+`afinfo {{file}}`
+
+- Print a one line description of the audio file:
+
+`afinfo -b {{file}}`
+
+- Print metadata info AND contents of the audio file's InfoDictionary:
```
Comment (einverne) [inline pages/osx/afinfo.md]:
```suggestion
- Print metadata info and contents of the audio file's InfoDictionary:
```

File: pages/osx/afinfo.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# afinfo
+
+> Audio file metadata parser for OS X.
+> `afinfo --help` or `afinfo -h` for full usage.
+
+- Display info of given audio file:
```
Comment (einverne) [inline pages/osx/afinfo.md]:
```suggestion
- Display info of a given audio file:
```

File: pages/osx/afinfo.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`afinfo {{file}}`
```
Comment (einverne) [inline pages/osx/afinfo.md]:
```suggestion
`afinfo {{path/to/file}}`
```
And change the followings

File: pages.zh/osx/afinfo.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# afinfo
+
+> 显示音频文件元数据 (Metadata) 详细信息 (OS X)
```
Comment (einverne) [inline pages.zh/osx/afinfo.md]:
```suggestion
> 显示音频文件元数据 (Metadata) 详细信息 (OS X).
```

We usually add a `.` at the end of the line. Although it is not the Chinese period.

File: pages/osx/afinfo.md (hunk 2 above)
Comment (garily) [reply pages/osx/afinfo.md]: This is a built-in command provided by OS X, I have updated the description.

**Comment by zdroid** [comment]
@garily I merged this nevertheless, but I'm curious which dialects of Chinese are the three provided by you and is this some kind of norm in documentation?

**Comment by garily** [comment]
Thanks for the merging!

Well I think I only used one dialect, i.e., Chinese Mandarin (the official language used by Mainland China, official locale being `zh-CN`, a variant of in Chinese Simpplified (`zh-Hans`, previously `zh-CHS`)).

Meanwhile, Hong Kong, Macau and Taiwan uses `zh-HK`, `zh-MO`, and `zh-TW` respectively, which are all variants of Chinese Traditional (`zh-Hant`, previously `zh-CHT`).

Singapore also uses `zh-SG`, also a variant of `zh-Hans`.

_While some elements of these character sets differ from their counterparts, people using `zh` character sets can normally mutually understand each other._ (However, they still have preferences 😀).

**Comment by zdroid** [comment]
Thanks for explanation, I got it. :rocket:

**Pull request merged**